scripts/vim-patch.sh

```diff
@@ -64,17 +64,17 @@ assign_commit_details() {
   if [[ ${1} =~ [0-9]\.[0-9]\.[0-9]{3,4} ]]; then
     # Interpret parameter as version number.
     vim_version="${1}"
-    vim_commit="v${1//./-}"
+    vim_commit="v${1}"
     local strip_commit_line=true
-    vim_commit_url="https://github.com/vim/vim/commit/${vim_commit}"
+    vim_commit_url="https://github.com/vim/vim/tree/${vim_commit}"
```

Comment:
Not sure why this doesn't work with the Vim repo, e.g. https://github.com/neovim/neovim/commit/nightly still works fine. But maybe it would be better to just use the commit hash for the URL, same as for non-tagged commits/runtime updates? I think it's very convenient to immediately see the changes with one click.